src/txdb.cpp

```diff
@@ -28,6 +28,10 @@ static const char DB_FLAG = 'F';
 static const char DB_REINDEX_FLAG = 'R';
 static const char DB_LAST_BLOCK = 'l';
 
+//! Used to cache the nChainTx value for the base block of a UTXO snapshot.
+//! This only contains a value for chainstates created based on a snapshot.
```

Comment:
In commit "txdb: add nChainTx cache for snapshot chainstates" (bc622c33ce925159906e3b00be77b11d598397ed)

"Only contains a value" suggests the row may be present without a value. Would suggest saying something like "This is not present in the normal coins database. It is only saved and used in in temporary UTXO snapshots."